source/server/admin/admin.cc

```diff
@@ -299,8 +299,8 @@ void AdminImpl::addCircuitSettings(const std::string& cluster_name, const std::s
 // TODO(efimki): Add support of text readouts stats.
 void AdminImpl::writeClustersAsJson(Buffer::Instance& response) {
   envoy::admin::v3::Clusters clusters;
-  for (auto& cluster_pair : server_.clusterManager().clusters()) {
-    const Upstream::Cluster& cluster = cluster_pair.second.get();
+  for (const auto& [_, cluster_opt] : server_.clusterManager().clusters()) {
```

Comment:
I am a little confused on what does "opt" stand for? As far as I know, the value here should be the reference wrapper of a cluster. Maybe cluster_ref is a better option?

Reply:
Gread idea, my bad, it should be a cluster_ref